### ISSUE TITLE
assess the inclusion of peer results example data

### ISSUE THREAD

**Issue by cjyetman** [RMI-PACTA/r2dii.interactive#23]
We currently include peer result data to drive the `peer_comparison_chart()` example. We should re-assess if/how we do that because I doubt we should be including real peer data.

Thoughts? @AlexAxthelm @jdhoffa @MonikaFu

**Comment by MonikaFu** [comment]
Agreed. Ideally we'd like to have example PACTA analysis output to use as examples for all charts I'd say.

**Comment by jdhoffa** [comment]
I would love to go through a big round of faking MOST datasets. Data prep outputs, PACTA outputs, everything. One day :-)

**Comment by jdhoffa** [comment]
Could even automate it,  so the same inputs fake data generates all the downstream fake data, and if we want to update it we just run it again on latest.
